livre/eq/Config.cpp

```diff
@@ -94,14 +100,27 @@ class Config::Impl
         , eventMapper( EventHandlerFactoryPtr( new EqEventHandlerFactory ))
         , redraw( true )
         , dataFrameRange( INVALID_FRAME_RANGE )
+        , dataSourceRange( std::numeric_limits<uint8_t>::min(), // This will change with the new TF
+                           std::numeric_limits<uint8_t>::max( ))
```

Comment:
This means the default range is [0, 255.0]? Why not writing it explicitly instead of using std::numeric_limits, it's more obvious that way.